GetRandomWordAPI/GetRandomWordAPI/Controllers/GetRandomWordController.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GetRandomWordAPI.Service;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GetRandomWordAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GetRandomWordController : ControllerBase
+    {
+        private GetRandomWordService GetRandomWordService;
+
+        public GetRandomWordController(GetRandomWordService getRandomWordService)
+        {
+            this.GetRandomWordService = getRandomWordService;
+        }
+        // GET: api/GetRandomWord
+        [HttpGet("/all")]
+        public List<string> GetAllWords()
+        {
+            return GetRandomWordService.GetAllWords();
+        }
+
+        // GET: api/GetRandomWord/5
+        [HttpGet("/word")]
+        public IEnumerable<string> GetWord(int number)
+        {
+            return GetRandomWordService.GetWord(number);
+        }
+        
+    }

```

Comment:
I am missing a "GetRandomWord" function here.

Reply:
with GetWord function if you don't give a number/amount you get a random word

Reply:
But you get "RandomWords" not just "Words" it ist just simply the naming that throwing me off here ;)